issue found on: analyser/aggregator.go:194:1: paramTypeCombine

an issue has been detected in source code: ./analyser/aggregator.go:194:1: paramTypeCombine: func(colorizer aurora.Aurora, entries []AggregatorLogEntry, notRead []AggregatorLogEntry) could be replaced with func(colorizer aurora.Aurora, entries, notRead []AggregatorLogEntry)

--- a/analyser/aggregator.go
+++ b/analyser/aggregator.go
@@ -197,7 +197,7 @@ func printConsumedEntries(colorizer aurora.Aurora, entries, notRead []Aggregator
 	fmt.Println()
 }
 
-func printReadEntries(colorizer aurora.Aurora, entries []AggregatorLogEntry, notRead []AggregatorLogEntry) {
+func printReadEntries(colorizer aurora.Aurora, entries, notRead []AggregatorLogEntry) {
 	for i, entry := range notRead {
 		printReadEntry(colorizer, i+1, entry)
 		printMessageForErrorsMessageWithOffset(colorizer, entries, entry.Offset)

